auth/auth.go

```diff
@@ -873,7 +873,7 @@ func (auth *Authenticator) authenticateJWTIdentity(identity *Identity, provider
 		base.InfofCtx(auth.LogCtx, base.KeyAuth, "Error retrieving OIDCUsername: %v", err)
 		return nil, PrincipalConfig{}, time.Time{}, err
 	}
-	base.DebugfCtx(auth.LogCtx, base.KeyAuth, "OIDCUsername: %v", base.UD(username))
+	base.DebugfCtx(auth.LogCtx, base.KeyAuth, "Got username %q from JWT", base.UD(username))
```

Comment:
It seems uncommon to have multiple providers, but should we be logging provider here?

I do not have a preference.